sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/models/AmqpMessageId.java

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp.models;
+
+import java.util.Objects;
+
+/**
+ * This represents Message id. Amqp specification support message id in various types. This class only implements
+ * {@link String} representation at present.
+ *
+ * <p><strong>Create and retrieve message id </strong></p>
+ * {@codesnippet com.azure.core.amqp.models.AmqpMessageId.createAndGet}
+ *
+ * <b>Amqp message id types:</b>
+ * <ul>
+ * <li><a href="http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#type-message-id-string" target="_blank">String</a></li>
+ * <li><a href="http://docs.oasis-open.org/amqp/core/v1.0/amqp-core-messaging-v1.0.html#type-message-id-ulong" target="_blank">long</a></li>
+ * <li><a href="http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#type-message-id-uuid" target="_blank">Uuid</a></li>
+ * <li><a href="http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#type-message-id-binary" target="_blank">binary</a></li>
+ * </ul>
```

Comment:
If this class only supports string id, do we need to have all other types in this javadoc?

Reply:
ok removing